miqi/skills/qraft-workflowspec-export/scripts/upload_run.py

```
@@ -0,0 +1,205 @@
+        self.raw = raw
+
+
+def pre_check(file_path: Path) -> dict[str, Any]:
+    """前置校验。返回解析后的 JSON；失败抛 UploadError（exit 2）。"""
+    if not file_path.is_file():
+        raise UploadError("PRECHECK_FAILED", f"文件不存在：{file_path}")
+    if file_path.suffix.lower() != ".json":
+        raise UploadError("PRECHECK_FAILED", "仅支持 .json 文件")
+    size = file_path.stat().st_size
+    if size > MAX_FILE_BYTES:
+        raise UploadError("PRECHECK_FAILED", f"文件 {size} 字节，超过 5MB 上限")
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)